### DISCUSSION TITLE
.sdr folder

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8155] [General]
I'm using an android tablet Samsung galaxy Tab A (T-295) with the latest koreader version 8.2021..
Every pdf file I open using koreader is creating a folder with the same file name and .sdr extension. It contains a (Metadata.pdf.lua) file, I think it's the book information like statistics or something but the problem is I have a folder (my library) it contains hundreds of pdfs, so my library folder now is full of folders with the same name of my pdfs.
I think this (Metadata) folders should be gathered in a single folder or it can be hidden by default.

My folder (library) is on SD card if that makes any difference, and I didn't notice this in the earlier versions (maybe 6 or 5.2021) and maybe I'm wrong just didn't notice.

![IMG_20210830_220359_522](https://user-images.githubusercontent.com/70109540/131392587-cd698386-7555-4bf8-8a66-103eebcd7ac4.jpg)

**Comment by mergen3107** [comment]
> or it can be hidden by default.

If you browse within KOReader's File Manager, they are hidden.

The .sdr directory is very important for file processing, so it should always be there once you open a book (or even if you mark a book "Read", "On hold", or "Finished"). I know for sure that each book must have it own .sdr directory (because it contains its own settings), but I don't know whether it is possible to have them collected in one place.

**Reply by Zeyadas** [reply]
Yes, using the koreader file browser will not show the .sdr folder, but usually I use a simple android file browser (ES file browser) it's easier and faster than the koreader's file browser.

**Comment by Inc0n** [comment]
I think for most ebook devices, such as a kindle, the file directory is hidden to the user for most of the time, however, for the use case of an android phone it may become be clustering the file system, a lot more visibly.

**Comment by Ketrel** [comment]
I have the same problem/annoyance with this.  

It's be far better if it used the koreader folder as a base, so if I have a book at say `/storage/emulated/0/Books/a/b/c/d.epub` the metadata folder would be `/storage/emulated/0/koreader/metadata/a/b/c/d.sdr/`

That seems like it'd be the best option to avoid filesystem clutter while keeping code changes to the minimum.

**Reply by poire-z** [reply]
Current discussions about this in https://github.com/koreader/koreader/issues/9265
